concrete/tools/permissions/access/entity/types/submitter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+defined('C5_EXECUTE') or die("Access Denied.");
+use Concrete\Core\Permission\Access\Entity\SubmitterEntity as SubmitterPermissionAccessEntity;
+
+if (\Loader::helper('validation/token')->validate('process')) {
+    $js = \Loader::helper('json');
+    $obj = new \stdClass();
```

Comment:
Since we are not in a namespace (there's no `namespace` at the beginning of the file), we can use root classes without the `\` prefix.
So, `new stdClass();` instead of `new \stdClass();`